test/unit/spec/MfaVerify_spec.js

```diff
@@ -174,6 +174,39 @@ function (Okta,
       });
     }
 
+    function setupWebauthnOnly() {
```

Comment:
Can you explain the reason why we need to break this out into a separate function (which looks very similar to the other setup function)?